lib/cpp/src/thrift/TApplicationException.h

```diff
@@ -21,6 +21,7 @@
 #define _THRIFT_TAPPLICATIONEXCEPTION_H_ 1
 
 #include <thrift/Thrift.h>
+#include <stdint.h>
```

Comment:
Still perplexing me... it looks like _STDINT is already defined by the time we get there but only stdint.h seems to define that... so I'm still looking...